hr/time-off.md

```diff
@@ -0,0 +1,51 @@
+
+## Goals
+
+- Normalize our PTO policies across countries, so that team members don’t get different PTO depending on their country of residence.
+- Choose PTO policies that minimize the chance that a team member will take *fewer- days off per year than the minimum in their country.
+- Do not apply hard limits and restrictions, but instead encourage soft minimums and coordination amongst team members.
+- Encourage a focus on contributions to the team and organization over number of days worked.
+- Base our numbers off of objective data where possible.
+
+## Policy
+
+- 2i2c does not have a limit on the number of days you may take off each year.
+- We recommend that all 2i2c team members take **at least 40 days** (8 weeks) annually (pro-rated if you started work mid-way through the year).[^time-off]
+- We do not treat holidays differently from vacation days.
+- You must open an issue to tell others about your intention to take time off, and you must add your intended time off to the [Team Leave Calendar](https://calendar.google.com/calendar/u/2?cid=Y19pNTJqZGNhbTZ0M3FsaDF1NTNqdG42MjNwY0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
+
+### Guidelines for taking time off
+
+- **Don’t ask, do tell.** Tell team members as early as possible before you take time off. A good rule of thumb is to tell others in advance twice as long as you’ll be off (so for 2 days, let people know 4 days in advance; for 1 week, let others know 2 weeks in advance, etc).
+- **Be mindful of the team.** We strive to build team processes that give us the capacity for team members to regularly take time off, but please consider the team’s obligations around the time you’re taking off. For example, during important team meetings, particularly high-stress events, etc.
+- **Discuss with others if there are objections.** If a team member believes that your time off would be unfairly harmful to the team, they may object via commenting in the github issue for your time off. In general, team members should only object if a person’s time off would significantly harm 2i2c’s ability to accomplish its mission. If there are objections, we should aim to resolve these via discussion amongst the team. However, ultimately it is the decision of the person requesting time off.
+- **Weekends and holiday seasons are exceptional.** We do not expect team members to work during weekends, and we also expect that many team members may be off simultaneously during heavy holiday seasons (in particular, the end of the year). This may create tension, as we are running services that run 24/7. We signal our intent to protect holidays and weekends by advertising reduced responsiveness with no promises during these times. If we feel obligated to engage with a community “outside work hours”, we should recognize this and ensure the people doing this work feel valued and fairly treated.
```

Comment:
```suggestion
- **Weekends and holiday seasons are exceptional.** We do not expect team members to work during weekends, and we also expect that many team members may be off simultaneously during heavy holiday seasons (in particular, the end of the calendar year). This may create tension, as we are running services that run 24/7. We signal our intent to protect holidays and weekends by advertising reduced responsiveness with no promises during these times. If we feel obligated to engage with a community “outside work hours”, we should recognize this and ensure the people doing this work feel valued and fairly treated.
```